server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -98,7 +98,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongAsyncCounter(
-                "es.node.stats.indices.merge.total",
+                "es.indices.merge.total",
                 "Total number of merge operations.",
                 "operation",
                 () -> new LongWithAttributes(stats.getOrRefresh().getIndices().getMerge().getTotal())
```

Comment:
btw. there is a lot of null pointers in apm agent logs. We should fix those nested getters..
```

{"@timestamp":"2023-12-14T13:02:24.448Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.fs.io_stats.io_time.total, description=The total time in millis spent performing I/O operations across all devices used by Elasticsearch., unit=milliseconds, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.monitor.fs.FsInfo$IoStats.getTotalIOTimeMillis()\" because the return value of \"org.elasticsearch.monitor.fs.FsInfo.getIoStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.monitor.fs.FsInfo$IoStats.getTotalIOTimeMillis()\" because the return value of \"org.elasticsearch.monitor.fs.FsInfo.getIoStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.449Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.coordinating_operations.memory.size.total, description=Total number of memory bytes consumed by coordinating operations, unit=bytes, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalCoordinatingBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalCoordinatingBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.449Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.coordinating_operations.count.total, description=Total number of coordinating operations, unit=operations, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalCoordinatingOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalCoordinatingOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.449Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.coordinating_operations.memory.size.current, description=Current number of memory bytes consumed by coordinating operations, unit=bytes, type=OBSERVABLE_GAUGE, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentCoordinatingBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentCoordinatingBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:54)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:51)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:56)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:53)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.450Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.coordinating_operations.count.current, description=Current number of coordinating operations, unit=operations, type=OBSERVABLE_GAUGE, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentCoordinatingOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentCoordinatingOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:54)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:51)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:56)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:53)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.450Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.coordinating_operations.rejections.total, description=Total number of coordinating operations rejections, unit=operations, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCoordinatingRejections()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCoordinatingRejections()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.450Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.primary_operations.memory.size.total, description=Total number of memory bytes consumed by primary operations, unit=bytes, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalPrimaryBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalPrimaryBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.450Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.primary_operations.count.total, description=Total number of primary operations, unit=operations, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalPrimaryOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getTotalPrimaryOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.451Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.primary_operations.memory.size.current, description=Current number of memory bytes consumed by primary operations, unit=bytes, type=OBSERVABLE_GAUGE, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentPrimaryBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentPrimaryBytes()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:54)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:51)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:56)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:53)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.451Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.primary_operations.count.current, description=Current number of primary operations, unit=operations, type=OBSERVABLE_GAUGE, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentPrimaryOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getCurrentPrimaryOps()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:54)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:51)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:56)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:53)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.451Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.primary_operations.rejections.total, description=Total number of primary operations rejections, unit=operations, type=OBSERVABLE_COUNTER, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getPrimaryRejections()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getPrimaryRejections()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:66)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongCounterBuilder$1.accept(BridgeLongCounterBuilder.java:63)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:64)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongCounterBuilder$1.accept(ProxyLongCounterBuilder.java:61)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
{"@timestamp":"2023-12-14T13:02:24.451Z","log.level": "WARN","message":"An exception occurred invoking callback for CallbackRegistration{instrumentDescriptors=[InstrumentDescriptor{name=es.node.stats.indices.indexing.memory.limit.current, description=Current memory limit for primary and coordinating operations, unit=bytes, type=OBSERVABLE_GAUGE, valueType=LONG, advice=Advice{explicitBucketBoundaries=null, attributes=null}}]}.", "ecs.version": "1.2.0","event.dataset":"elasticsearch.apm-agent","process.thread.name":"elastic-apm-otel-metrics-exporter","log.logger":"io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration","error.type":"java.lang.AssertionError","error.message":"observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getMemoryLimit()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]","error.stack_trace":"java.lang.AssertionError: observer must not throw [Cannot invoke \"org.elasticsearch.index.stats.IndexingPressureStats.getMemoryLimit()\" because the return value of \"org.elasticsearch.action.admin.cluster.node.stats.NodeStats.getIndexingPressureStats()\" is null]\n\tat org.elasticsearch.telemetry.apm@8.13.0-SNAPSHOT/org.elasticsearch.telemetry.apm.internal.metrics.OtelHelper.lambda$longMeasurementCallback$2(OtelHelper.java:79)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:54)\n\tat co.elastic.apm.agent.opentelemetry.metrics.bridge.v1_14.BridgeLongGaugeBuilder$1.accept(BridgeLongGaugeBuilder.java:51)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:56)\n\tat co.elastic.apm.agent.embeddedotel.proxy.ProxyLongGaugeBuilder$1.accept(ProxyLongGaugeBuilder.java:53)\n\tat io.opentelemetry.sdk.metrics.AbstractInstrumentBuilder.lambda$registerLongAsynchronousInstrument$1(AbstractInstrumentBuilder.java:119)\n\tat io.opentelemetry.sdk.metrics.internal.state.CallbackRegistration.invokeCallback(CallbackRegistration.java:84)\n\tat io.opentelemetry.sdk.metrics.internal.state.MeterSharedState.collectAll(MeterSharedState.java:96)\n\tat io.opentelemetry.sdk.metrics.SdkMeter.collectAll(SdkMeter.java:75)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$LeasedMetricProducer.produce(SdkMeterProvider.java:193)\n\tat io.opentelemetry.sdk.metrics.SdkMeterProvider$SdkCollectionRegistration.collectAllMetrics(SdkMeterProvider.java:217)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.doRun(PeriodicMetricReader.java:161)\n\tat io.opentelemetry.sdk.metrics.export.PeriodicMetricReader$Scheduled.run(PeriodicMetricReader.java:153)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\n\tat java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:358)\n\tat java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat co.elastic.apm.agent.util.ExecutorUtils$2.run(ExecutorUtils.java:99)\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\n"}
```
@piergm we missed that when the PR was merged. Is this something you can help fixing?

Reply:
Looking into it!

Reply:
Can you point to where to find these logs? I am trying to understand if this happens at startup or due to some other event during the node lifecycle.
A quick fix, that does not fix the root cause is to either try catch or use optionals. For this particular metric I'd do:
```
() -> new LongWithAttributes(Optional.ofNullable(stats.getOrRefresh())
    .map(a -> a.getIndices()))
    .map(b -> b.getMerge())
    .map(c -> c.getTotal())
    .orElse(0));
```
But this can be a fix only if the problem is at startup. If instead it occurs during the lifecycle of the node is not correct to set a metric to 0.

Reply:
when running with apm agent enabled look for apm_agent.json file in logs directory `build/testclusters/runTask-0/logs`

Reply:
I suspect this null issue should be fixed by https://github.com/elastic/elasticsearch/pull/103520